src/config.rs

```diff
@@ -166,6 +167,29 @@ impl IndexMut<usize> for ColorList {
     }
 }
 
+#[derive(Debug, Deserialize)]
+pub struct Shell<'a> {
+    program: Cow<'a, str>,
+    args: Vec<String>,
```

Comment:
In your config, omit `shell.args` and it should still work. It currently gives an error about the field missing.